.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: 'bundler'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```

Comment:
Hopefully not too noisy given the few dependencies

Reply:
yeah, we can always switch to weekly if needed!